ui/src/plugins/fetch.js

```diff
@@ -38,3 +40,7 @@ export function fetchWithContext(
       }
     });
 }
+
+export function cleanDuplicateSlash(path) {
+  return path.replace(/([^:]\/)\/+/g, "$1");
+}
```

Comment:
This file is missing new line at the end of file too.

Reply:
Done